javascript/index.js

```diff
@@ -40,26 +42,69 @@ function clearSplits() {
 
 function setStopBtn() {
   // ... your code goes here
+  /*set*/ 
+    /*set*/ 
+
+ /*set*/ 
 }
 
 function setSplitBtn() {
   // ... your code goes here
+  
 }
 
 function setStartBtn() {
   // ... your code goes here
+  /*set*/
 }
 
 function setResetBtn() {
   // ... your code goes here
 }
 
 // Start/Stop Button
-btnLeftElement.addEventListener('click', () => {
+
   // ... your code goes here
-});
+  btnLeftElement.addEventListener('click', () => {
+
+  if(btnLeftElement.textContent === 'START' ) {
+    btnLeftElement.textContent = 'STOP';
+    btnRightElement.textContent = 'SPLIT';
+    chronometer.start(callback);
```

Comment:
the word "callback" is only a placeholder (in chronometer.js) so here you need to pass a function that you actually wrote (--> printTime)